src/FixedDataTableRow.js

```diff
@@ -248,7 +248,7 @@ class FixedDataTableRowImpl extends React.Component {
       'public/fixedDataTableRow/fixedColumnsDivider': left > 0,
       'public/fixedDataTableRow/columnsShadow': this.props.scrollLeft > 0,
      });
-     // Subtract 1 for the border
+     // Subtract 1 to account for .fixedDataTableLayout/header border-bottom-width
```

Comment:
Here is the comment for the magic 1px.

Reply:
Ah true, you're right.

I still prefer a const, but this is fine :stuck_out_tongue_closed_eyes:

Reply:
I don't mind, since you prefer it (I have no strong preference) - I'll define HEADER_BORDER_BOTTOM_WIDTH and include the description from this comment above it :)

Reply:
sweet, thanks